config/routes.rb

```diff
@@ -13,5 +13,5 @@
 
   resources :categories
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  root 'home#index'
```

Comment:
Is this related to the DB migration?

Reply:
Thanks for catching that. It should go in my next pr instead. Should be fixed now.